frontend/__tests__/podcast.test.jsx

```diff
@@ -0,0 +1,10 @@
+import '@testing-library/jest-dom'
+import { formatPodcastDate } from "../lib/helpers";
+
+describe('Podcast', () => {
+  it('podcast date formatting', () => {
```

Comment:
Structure test cases like the following: 

```suggestion
  describe('Podcast', () => {
      describe('podcast date formatting', () => {
      // replace it statement content with the relevant test cases 
         it('formats same month and date to equal April 11, 2022', () => {
            // expect code here
         }
      }
  }
```